.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js: "12"
 
-script: yarn lint
```

Comment:
Do you also want to run lint without the `--fix` flag?

Reply:
My understanding is that `yarn test` is the same thing as `npm run test`, which will run `jest && standard` returning an error if either are unsuccessful.

Reply:
Oh I missed that that script was doing 2 things...